### DISCUSSION TITLE
Question about multiple plow workers on same field

### DISCUSSION THREAD

**Discussion by bojanh66** [Courseplay/courseplay#6788] [Q&A] [answered]
Developers, is it possible to make a course which will do plowing like in real life ?
Let me show you example in this video:
https://www.youtube.com/watch?v=IcWuVXUvj9w&t

If you notice, the 1st tractor always wait for 2nd and 3rd tractor to finish the furrow before he starts a new furrow.
I was playing/experimenting with CP, but i could not manage to make a such course.
I know i can make multiple tools in Course generator but thats it, i dont know how to make 1st tractor wait for other tractors to finish furrow before starting new furrow...is it even possible?
This is merely for realistic purpose, i am trying to create realistic course.

**Comment by pvaiko** [comment, marked as answer]
Wow, that's beautiful. And no, we can't do that:
1. this would need a new turn type (doing this figure 8), there's a similar request in our queue here: #5812
2. we would have to implement something to make multi tools follow each other much closer. This would be the most difficult I think.
